packages/plugins/output/src/validate.test.ts

```diff
@@ -0,0 +1,117 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the MIT License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2019-Present Datadog, Inc.
+
+import { validateOptions } from './validate';
+
+describe('validateOptions', () => {
+    describe('enable', () => {
+        const cases = [
+            {
+                description: 'return false when no output config provided',
+                input: {},
+                expected: false,
+            },
+            {
+                description: 'return true when output config is an empty object',
+                input: { output: {} },
+                expected: true,
+            },
+            {
+                description: 'return true when output config has enable: true',
+                input: { output: { enable: true } },
+                expected: true,
+            },
+            {
+                description: 'return false when output config has enable: false',
+                input: { output: { enable: false } },
+                expected: false,
+            },
+        ];
+
+        test.each(cases)('Should $description', ({ input, expected }) => {
```

Comment:
Is `$description` some magic feature of test runner?

Reply:
I never saw this; googling around I found some [blogs](https://www.jacobparis.com/content/dynamically-generate-test-cases) and StackOverflows telling that it should be `%` not `$`

Reply:
From the [jest docs](https://jestjs.io/docs/api#1-describeeachtablename-fn-timeout)

> Or generate unique test titles by injecting properties of test case object with $variable
To inject nested object values use you can supply a keyPath i.e. $variable.path.to.value (only works for "own" properties, e.g. $variable.constructor.name wouldn't work)
You can use $# to inject the index of the test case
You cannot use $variable with the printf formatting except for %%